### ISSUE TITLE
Template from JS

### ISSUE THREAD

**Issue by georges-gomes** [github/catalyst#82]
Hi

I'm working on a Catalyst template for webcomponents.dev

I built this one
https://webcomponents.dev/edit/YclS8tjAV7ea8I87DIMG

For future reference this is the code:
```
import { controller, target } from "@github/catalyst";

const template = document.createElement("template");
template.innerHTML = `<style>
    * {
      font-size: 200%;
    }

    span {
      width: 4rem;
      display: inline-block;
      text-align: center;
    }

    button {
      width: 4rem;
      height: 4rem;
      border: none;
      border-radius: 10px;
      background-color: seagreen;
      color: white;
    }
  </style>
  <button data-action="click:my-counter#dec">-</button>
  <span data-target="my-counter.value"></span>
  <button data-action="click:my-counter#inc">+</button>`;

@controller
class MyCounterElement extends HTMLElement {
  @target value: HTMLElement;

  count: number = 0;

  connectedCallback() {
    this.attachShadow({ mode: "open" });
    this.shadowRoot.appendChild(template.content.cloneNode(true));
    this.update();
  }

  inc() {
    this.count++;
    this.update();
  }

  dec() {
    this.count--;
    this.update();
  }

  update() {
    this.value.textContent = "" + this.count;
  }
}
```

Visuals are ok.
but clicks don't work.

If I remove the shadowroot then clicks work.

Note: I tried to use `@github/jtml` referenced in the documentation but it's not open to public yet (?)

Thanks
Georges

cc @koddsson

**Comment by keithamus** [comment]
Fixes in `v1.0.2` :tada:

**Comment by georges-gomes** [comment]
Thanks!

And I made it work with `jtml` as well : https://webcomponents.dev/edit/YclS8tjAV7ea8I87DIMG

👍
